modules/tools/calibration/process_data.sh

```diff
@@ -18,8 +18,9 @@
 
 #! /bin/bash
 
-rm result.csv
-for f in `ls ${1}/*_recorded.csv`
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}"  )" && pwd  )"
```

Comment:
Remove redundant spaces.

Reply:
I have already modified them, please review them again. Think you!